bin/opx-config-interface

```diff
@@ -14,6 +14,7 @@
 # See the Apache Version 2.0 License for specific language governing
 # permissions and limitations under the License.
 
+from __future__ import print_function
```

Comment:
So is it meant to make the change forwards compatible with Python3 ?

We have to make many other changes to the code, so let's make them all at the same time, rather than using this approach.

Reply:
It is actually a syntax error in the print line referred from #26 . Correct syntax would be 
```print >> sys.stderr, 'Failed to set port {}'.format(port) ```
But this syntax is not valid for python3, so made this change just to be easier portable.